Creating posix group on remote 389ds instance fails using lib389

**Issue Description**
When attempting to create a posix group on a remote 389ds instance a failure occurs

**Package Version and Platform:**
 - Platform: ubuntu
 - Package and version: 
    ```
    apt show python3-lib389
    Package: python3-lib389
    Version: 2.0.15-1
    Priority: optional
    Section: universe/net
    Source: 389-ds-base
    Origin: Ubuntu
    Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
    Original-Maintainer: Debian FreeIPA Team <pkg-freeipa-devel@alioth-lists.debian.net>
    Bugs: https://bugs.launchpad.net/ubuntu/+filebug
    Installed-Size: 2196 kB
    Depends: python3-argcomplete, python3-argparse-manpage, python3-dateutil, python3-distro, python3-ldap, python3-pkg-resources, python3-pyasn1, python3-pyasn1-modules, python3:any, libnss3-tools, openssl, python3-packaging, python3-pytest
    Conflicts: 389-ds-base (<< 1.4.0.18-1~), python-lib389 (<< 1.3.7.8)
    Replaces: 389-ds-base (<< 1.4.0.18-1~), python-lib389 (<< 1.3.7.8)
    Homepage: https://directory.fedoraproject.org
    Download-Size: 366 kB
    APT-Manual-Installed: yes
    APT-Sources: http://mirror.aarnet.edu.au/pub/ubuntu/archive/ jammy/universe amd64 Packages
    Description: Python3 module for accessing and configuring the 389 Directory Server
    This Python3 module contains tools and libraries for accessing, testing,
    and configuring the 389 Directory Server.
    ```
 - Browser [e.g. chrome, safari] N/A

**Steps to Reproduce**
Steps to reproduce the behavior:
1. Install python3-lib389
   ```
   apt install python3-lib389
   ```
2. Start and initialize 389ds instance in docker
   ```
   docker run --rm -d -p 636:3636 -e DS_DM_PASSWORD=ldapdmpassword --name 389ds 389ds/dirsrv:latest
   docker exec -i -t 389ds dsconf slapd-localhost backend create --suffix dc=test,dc=com --be-name userRoot
   docker exec -i -t 389ds dsidm slapd-localhost -b dc=test,dc=com initialise
   ```
3. Run the following python script
   ```
       from lib389 import DirSrv
    from lib389.idm.posixgroup import PosixGroups

    ds = DirSrv()

    ds.remote_simple_allocate(ldapuri='ldaps://localhost:636', binddn='cn=Directory Manager', password='ldapdmpassword')
    ds.open(connOnly=True, reqcert=False)

    groups = PosixGroups(ds, 'dc=test,dc=com')

    groups.create(properties={'cn': 'asdf', 'gidNumber': '10000'})
   ```
4. See error
   ```
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
      File "/usr/lib/python3/dist-packages/lib389/_mapped_object.py", line 1223, in create
        co = self._entry_to_instance(dn=None, entry=None)
      File "/usr/lib/python3/dist-packages/lib389/_mapped_object.py", line 1047, in _entry_to_instance
        return self._childobject(instance=self._instance, dn=dn)
      File "/usr/lib/python3/dist-packages/lib389/idm/posixgroup.py", line 37, in __init__
        if not ds_is_older('1.3.7'):
      File "/usr/lib/python3/dist-packages/lib389/utils.py", line 1108, in ds_is_older
        return ds_is_related('older', *ver, instance=instance)
      File "/usr/lib/python3/dist-packages/lib389/utils.py", line 1093, in ds_is_related
        ds_ver = get_ds_version()
      File "/usr/lib/python3/dist-packages/lib389/utils.py", line 1083, in get_ds_version
        return paths.version
      File "/usr/lib/python3/dist-packages/lib389/paths.py", line 203, in __getattr__
        self._read_defaults()
      File "/usr/lib/python3/dist-packages/lib389/paths.py", line 150, in _read_defaults
        spath = self._get_defaults_loc(DEFAULTS_PATH)
      File "/usr/lib/python3/dist-packages/lib389/paths.py", line 147, in _get_defaults_loc
        raise IOError('defaults.inf not found in any well known location!')
   ```

**Expected results**
Posixgroup is successfully created

Regression found: test_schema_comparewithfiles CI test fails with:
>       if ds_is_newer('1.3.6.0', instance=self._instance):
E       AttributeError: 'SchemaLegacy' object has no attribute '_instance'

--- a/src/lib389/lib389/schema.py
+++ b/src/lib389/lib389/schema.py
@@ -567,7 +567,7 @@ class SchemaLegacy(object):
         """return a list of the schema files in the instance schemadir"""
         file_list = []
         file_list += glob.glob(self.conn.schemadir + "/*.ldif")
-        if ds_is_newer('1.3.6.0', instance=self._instance):
+        if ds_is_newer('1.3.6.0', instance=self.conn):
             file_list += glob.glob(self.conn.ds_paths.system_schema_dir + "/*.ldif")
         return file_list
 

